TeamPage.html

```diff
@@ -593,6 +593,48 @@ <h2>Meet Our Team</h2><br>
                </div>
            </div>
           </div>
+          <div class="col-sm-3 col-xs" style="min-width: 210px;">
+            <div class="id_presentation">
+              <img class="img_staff" src="https://i.imgur.com/jxzYjQO.jpg" style="width: 200px; height: 200px; object-fit: cover; object-position: top;" alt="Michelle's Picture"/>

```

Comment:
You might be able to center your picture a bit better if you remove the `object-position: top` CSS property here. But if this was intentional, let me know.